docs/HowTo/Voluntary-Exit.md

```diff
@@ -44,10 +48,9 @@ public key of the exiting validator:
 !!! example
 
    ```bash
-   teku voluntary-exit --beacon-node-api-endpoint=http://10.32.100.1:5051 \
+   teku voluntary-exit --beacon-node-api-endpoint=http://127.0.0.1:5051 \
    --validators-external-signer-public-keys=888eef... \
```

Comment:
As this is an example, having the full public key would be nice. I know this takes space, but if you really want the example to help users really need to be able to identify all the values and see if it looks like what they plan to use (for instance if the length is similar)